src/features/views/components/ViewDataTable/columnTypes/PersonFieldColumnType.tsx

```diff
@@ -0,0 +1,42 @@
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { IColumnType } from '.';
+import { PersonFieldViewColumn, ZetkinViewColumn } from '../../types';
+import { EnumChoice } from 'utils/types/zetkin';
+
+type SimpleData = string | number | boolean | null;
+
+export default class PersonFieldColumnType
+  implements IColumnType<ZetkinViewColumn, SimpleData>
+{
+  cellToString(): string {
+    return '';
+  }
+
+  private enumChoices: EnumChoice[] | null = null;
+
+  getColDef(column: PersonFieldViewColumn): Omit<GridColDef, 'field'> {
+    return {
+      filterable: true,
+      valueGetter: (params) => {
+        const cell = params.row[params.field];
+        if (column.config.enum_choices) {
+          this.enumChoices = column.config.enum_choices;
+          const choice = column.config.enum_choices.find((c) => c.key == cell);
```

Comment:
I do find this to be a bit weird, that `enum_choices` are part of the config even though they can't be edited. I know you said it's necessary, and maybe it is, but it doesn't feel right and it's something we should revisit when we define the schemas of API v2.

Reply:
We decided not to make any changes to this for now.